plugins/pyenv-virtualenvwrapper/pyenv-virtualenvwrapper.plugin.zsh

```diff
@@ -0,0 +1,51 @@
+eval "$(pyenv init -)"
+eval "$(pyenv-sh-virtualenvwrapper -)"
```

Comment:
Check that the command is available before loading.

Reply:
Also, any thoughts on using pyenv-sh-virtualenvwrapper_lazy as the default or at least allowing it as an option?